### PR TITLE
feat: added token, refreshToken cookieName option

### LINKED ISSUE #650: Invalid Cookie name
### Describe the bug

The cookie name being set as `auth:token` includes the separator `:` which is **invalid** as a cookie name.

[RFC 6256](https://www.rfc-editor.org/rfc/rfc6265#section-4.1.1) defines the cookie name as being a token:
> <token, defined in [[RFC2616], Section 2.2](https://www.rfc-editor.org/rfc/rfc2616#section-2.2)>

With RFC2616 stating that a token is `1*<any CHAR except CTLs or separators>`.

The remedy here would be to replace the usage of `:` with `_` which is an allowed character.

### Additional context

This problem came up when I was deploying a Nuxt project to aws using the aws cdk when creating a `CachePolicy` configured to pass through the `auth:token` cookie aws refused to complete citing the cookie name containing invalid characters.

### PR DESCRIPTION
**Pull request by FAL-coffee**
Closes #650 .

---
Allow the user to change the property name where tokens are stored in the cookie when using local or refresh types.

---
**Why this is needed:**
 Conflicts occur with where tokens are stored when multiple environments are launched on localhost.
It also solves the problem that issue #650 refers to.

---
Checklist:
- [x] issue number linked above after pound (`#`)
    - replace "Closes " with "Contributes to" or other if this PR does not close the issue
- [x] manually checked my feature / checking not applicable
- [ ] wrote tests / testing not applicable
- [ ] attached screenshots / screenshot not applicable

### CODE REVIEW

**Review by phoenix-ru — APPROVED** [review]

**Comment by phoenix-ru** [comment]
Looks good to me.
@zoey-kaiser OP is right in #650, should we use `auth.token` and `auth.refresh-token` as default cookie names instead?

**Comment by FAL-coffee** [comment]
> Looks good to me. @zoey-kaiser OP is right in #650, should we use `auth.token` and `auth.refresh-token` as default cookie names instead?

In my personal opinion, changing the default CookieName could pose a risk of logging users out across all existing services when updating the library version. 
Additionally, since restrictions only apply in certain cases, I believe it is unnecessary to change the default CookieName.

File: docs/content/2.configuration/2.nuxt-config.md [hunk 1]
```diff
@@ -193,6 +193,13 @@ type ProviderLocal = {
      * @example Beer
      */
     type?: string,
+    /**
+     * It refers to the name of the property when it is stored in a cookie.
+     *
+     * @default auth:token
```
Comment (phoenix-ru) [inline docs/content/2.configuration/2.nuxt-config.md:199]: Please, change `auth:token` to `auth.token` everywhere

File: src/module.ts [hunk 2]
```diff
@@ -75,12 +76,14 @@ const defaultsByBackend: {
...
       signInResponseTokenPointer: '/token',
       type: 'Bearer',
+      cookieName: 'auth:token',
       headerName: 'Authorization',
       maxAgeInSeconds: 5 * 60,
       sameSiteAttribute: 'none' // 5 minutes
     },
     refreshToken: {
       signInResponseRefreshTokenPointer: '/refreshToken',
+      cookieName: 'auth:refresh-token',
```
Comment (phoenix-ru) [inline src/module.ts:86]: Do the same for `auth:refresh-token`

**Review by phoenix-ru — CHANGES_REQUESTED** [review]

**Comment by phoenix-ru** [comment]
@FAL-coffee I have functionally tested your changes. Please, address the review and it can be safely merged.

**Comment by FAL-coffee** [comment]
@phoenix-ru 
I have committed the change to switch the default delimiter from ':' to '.'!

**Review by phoenix-ru — APPROVED** [review]

**Pull request merged**